clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerService.java

```diff
@@ -395,15 +395,24 @@ public long getMaxElapsedTimeMSSinceLastPollInConsumerPool() {
          * are zero-based and ConsumptionTasks are submitted to the executor in order.
          */
         Thread slowestThread = threadFactory.getThread(slowestTaskId);
-
+        SharedKafkaConsumer consumer = consumerToConsumptionTask.getByIndex(slowestTaskId).getKey();
+        Map<PubSubTopicPartition, TopicPartitionIngestionInfo> topicPartitionIngestionInfoMap =
+            getIngestionInfoFromConsumer(consumer);
+        // Convert Map of ingestion info for this consumer to String for logging with each partition line by line
+        StringBuilder sb = new StringBuilder();
```

Comment:
Nowaday's JVM can easily optimize string concatenate by simply using a stringBuilder for you. So String concatenation is not a sin anymore. 

Not an issue. Just an FYI.